### PR TITLE
Move the contact us map into the heading

### LINKED ISSUE #339: Move the contact us map into the heading
**Is your feature request related to a problem? Please describe.**
The contact page doesn't have a heading image. We can move the map into the heading. 

**Describe the solution you'd like**
ex: https://demos.creative-tim.com/material-kit-pro-bs4/examples/contact-us.html

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by Yoshitha-SACK**
## Purpose

The purpose of this PR is to fix #339

## Goals

Moving the contact us map into the heading

## Approach

The removed the map from current location and put it into the new location.

### Screenshots

image
image

  
### Preview Link
https://yoshitha-sack.github.io/sack-site/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [ ] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: contact_us/index.html [hunk 1]
```diff
@@ -34,10 +39,5 @@ <h3 class="font-weight-bold pb-3">Sports Complex</h3>
         <p>Tel: 081 221 3557</p>
       </div>
     </div>
-    <div class="container mt-4">
-      <div class="row">
-        <iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d5005.1003110759!2d80.62775051562666!3d7.322164927958504!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x79aa0d16eecbae25!2sSt.%20Anthony&#39;s%20College!5e0!3m2!1sen!2slk!4v1641754171541!5m2!1sen!2slk" width="100%" height="300vh" style="border:0;" allowfullscreen="" loading="lazy"></iframe>   
-      </div>
-    </div>
   </section>
 </div>
```
Comment (kumuditha-udayanga) [inline contact_us/index.html:43]: Add a new line to the end of the file.

File: assets/css/custom.css [hunk 2]
```diff
@@ -40,6 +40,12 @@
   color: #00316A;
 }
 
+.google-map {
+  position: relative;
+  top: 114px;
+  z-index: 99;
+}
+
```
Comment (jayasanka-sack) [inline assets/css/custom.css:48]: Wrap the page with a div with `.contact-us` card. 

```suggestion
.contact-us .google-map {
  position: relative;
  top: 114px;
  z-index: 99;
}

@media only screen and (max-width: 991px) {
  .contact-us .google-map {
    top: 123px;
  }
}
```

**Review by kumuditha-udayanga — CHANGES_REQUESTED** [review]

**Review by jayasanka-sack — APPROVED** [review]
Thanks @Yoshitha-SACK !

**Comment by Yoshitha-SACK** [comment]
Welcome

**Review by kumuditha-udayanga — APPROVED** [review]

**Comment by kumuditha-udayanga** [comment]
Thank you @Yoshitha-SACK

**Pull request merged**